src/components/RadioButton/RadioButtonInput.tsx

```diff
@@ -33,19 +33,13 @@ export const RadioButtonInput: VFC<Props> = ({ onChange, ...props }) => {
   )
 }
 
-const Wrapper = styled.span<{ themes: Theme }>`
-  ${({ themes }) => {
-    const { leading } = themes
-
-    return css`
-      position: relative;
-      display: inline-block;
-      flex-shrink: 0;
-      width: ${leading.NONE}rem;
-      height: ${leading.NONE}rem;
-      line-height: 1;
-    `
-  }}
+const Wrapper = styled.span`
+  position: relative;
+  display: inline-block;
+  flex-shrink: 0;
+  width: 1em;
+  height: 1em;
+  line-height: 1;
```

Comment:
imo： `line-height` はデザイントークンがある。

```suggestion
  line-height: ${leading.NONE};
```